examples/core/core_automation_events.c

```diff
@@ -71,7 +71,7 @@ int main(void)
     
     // Automation events
     AutomationEventList aelist = LoadAutomationEventList(0);  // Initialize list of automation events to record new events
-    SetAutomationEventList(&aelist);
+    SetAutomationEventList(aelist);
```

Comment:
This PR is malformed, `UnloadAutomationEventList()` is the changed function, not `SetAutomationEventList()`.